src/templates/apiIndexPage.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from 'gatsby';
+import PageTitle from '../components/PageTitle';
+import SEO from '../components/seo';
+import { Layout, Link } from '@newrelic/gatsby-theme-newrelic';
+import MDXContainer from '../components/MDXContainer';
+
+const ApiIndexPage = ({ data }) => {
+  const {
+    mdx: {
+      body,
+      frontmatter: { title },
+    },
+    allMdx: { nodes: apiDocPages },
+  } = data;
+  return (
+    <>
+      <SEO title={title} />
+      <PageTitle>{title}</PageTitle>
+      <Layout.Content>
+        <MDXContainer body={body}>
+          {apiDocPages.map((apiDoc) => {
+            return (
```

Comment:
We could probably return the JSX without the need for `return` here.

Reply:
ah gotcha. done!